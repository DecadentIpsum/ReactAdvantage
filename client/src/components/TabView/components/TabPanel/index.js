import React, { Component } from 'react';
import { TabPanel as TabPrimePanel } from 'primereact/components/tabview/TabView';

export default class TabPanel extends Component {
    render() {
        return (
            <TabPrimePanel {...this.props}>
                {this.props.children}
            </TabPrimePanel>
        );
    }
}
