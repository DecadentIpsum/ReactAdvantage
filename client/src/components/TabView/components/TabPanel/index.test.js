import React from 'react';
import Enzyme, { shallow } from 'enzyme';

import TabPanel from './index';

it('renders correctly', () => {
    const component = shallow(<TabPanel />);

    expect(component).toMatchSnapshot();
});
