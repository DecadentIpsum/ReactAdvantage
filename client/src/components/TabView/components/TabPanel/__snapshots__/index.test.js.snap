// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <TabPanel />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "children": undefined,
      "contentClassName": null,
      "contentStyle": null,
      "disabled": false,
      "header": null,
      "headerClassName": null,
      "headerStyle": null,
      "leftIcon": null,
      "rightIcon": null,
    },
    "ref": null,
    "rendered": null,
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": undefined,
        "contentClassName": null,
        "contentStyle": null,
        "disabled": false,
        "header": null,
        "headerClassName": null,
        "headerStyle": null,
        "leftIcon": null,
        "rightIcon": null,
      },
      "ref": null,
      "rendered": null,
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
